MetaMask/metamask-extension
# Fix #18423 - Ensure token details stay within dropdown

## Explanation

* Fixes #18423

Ensures that NFT details stay within the dropdown on the Send screen.

## Screenshots/Screencaps

<img width="552" alt="SCR-20230616-kmnr" src="https://github.com/MetaMask/metamask-extension/assets/46655/5db1a7e9-d130-4c35-bc4e-b83f220931da">


## Manual Testing Steps

You can see this visually at: http://localhost:6006/?path=/story/pages-send-sendcontent--nft-story

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
